examples/vue/src/pages/ui/components/authenticator/sign-in-federated/aws-exports.js

```diff
@@ -1,2 +1,2 @@
-import awsExports from '@environments/auth/auth-with-federated/src/aws-exports';
```

Comment:
As discussed offline, we won't be having a separate environment for `auth-with-federated`.